guides/15_Background_Filter_Video_Processor.md

```diff
@@ -141,6 +141,18 @@ if (await BackgroundReplacementVideoFrameProcessor.isSupported()) {
 let transformDevice = new DefaultVideoTransformDevice(logger, device, processors);
 ```
 
+### Changing the background image for Background Replacement
+
+If you would like to change the background image mid stream for a background replacement processor, you may want to cache a reference to the created BackgroundReplacementProcessor and call the [`setImageBlob`](https://github.com/aws/amazon-chime-sdk-js/blob/main/src/backgroundreplacementprocessor/BackgroundReplacementProcessor.ts#L28) API to change the background image.
```

Comment:
Suggest updating the wording to:
Create and store reference to the background replacement processor object. Use BackgroundReplacementProcessor's [setImageBlob](<link>) to update the existing background image within the BackgroundReplacementProcessor. For clarity, you can link how to create the processor on the "create" keyword.

Reply:
Sounds good, will update.

Reply:
I think having the word `create` might be a little confusing because it suggest creating a new processor. This section is about changing an already created processor background image. How about just starting the paragraph with `Store` ? thoughts?